src/oas3/__tests__/operation.test.ts

```diff
@@ -1,6 +1,72 @@
 import { transformOas3Operation } from '../operation';
 
 describe('transformOas3Operation', () => {
+  test('should return deprecated property', () => {
```

Comment:
I think this test is checking out a micro functionality from a too high level perspective (entire oas3 operation); I would just remove this.